### PR TITLE
Adding oracledb lib to mage-ai

### PR DESCRIPTION
**Pull request by Luishfs**
# Description

Added oracledb lib to mage-ai requirements.txt

# How Has This Been Tested?

Tested mage-ai build

# Checklist
- [X] The PR is tagged with proper labels (bug, enhancement, feature, documentation)
- [X] I have performed a self-review of my own code

cc:
@wangxiaoyou1993

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -24,6 +24,7 @@ jupyter_client==7.3.4
 ldap3==2.9.1
 newrelic==8.8.0
 numpy>=1.21.0
+oracledb==1.3.1
```
Comment (wangxiaoyou1993) [inline requirements.txt:27]: Could you add it to extra packages?
Also update setup.py https://github.com/mage-ai/mage-ai/blob/master/setup.py#L42

Comment (Luishfs) [reply requirements.txt]: Just added that on fb30b2cd00edfec210aec9086755bff2ca624359
Also did a `oracle` variable on setup.py

File: setup.py [hunk 2]
```diff
@@ -59,6 +59,9 @@ def readme():
         'clickhouse': [
             'clickhouse-connect==0.5.20',
         ],
+        'oracle': [
```
Comment (wangxiaoyou1993) [inline setup.py:62]: nit: alphabetical order

Comment (Luishfs) [reply setup.py]: Fixed on a01ced64437106d30e096e7230739d87d7b4e20c

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**